Sources/Classes/core/SecureStorage/Database/MiniAppSecureStorageDatabase.swift

```diff
@@ -0,0 +1,94 @@
+import Foundation
+
+public protocol MiniAppSecureStorageDatabase: AnyObject {
```

Comment:
Do we need to have this as `public` ?

Reply:
In LINK? Adding test cases in our SDK isn't enough? Just curios.
We need to limit exposing the Public interfaces/models unless its really needed.

Reply:
@rleojoseph Yeah I mean we can go without making it public but that caused quite some headaches in the past because we need to fulfill the 80% coverage on Link as well and no possibility to override or mock things.. I was thinking about making a bridge for the MiniApp SDK in Link as well which we ignore from coverage that would solve some issues. wdyt?

Reply:
Yeah. That would be great. Instead of exposing the entire class/protocol we can have a bridge. We can even add these files in code cov ignore list if we had written testcases in SDK.

I guess you can add the changelog for the storage mechanism and for README, we need to add the documentation set of interfaces which will exposed to Host app

Reply:
ok sounds good 👍  I'll remove the public modifiers and will prepare the bridge soon